src/intptrcast.rs

```diff
@@ -45,8 +45,10 @@ pub struct GlobalStateInner {
 }
 
 impl VisitTags for GlobalStateInner {
-    fn visit_tags(&self, _visit: &mut dyn FnMut(BorTag)) {
-        // Nothing to visit here.
+    fn visit_tags(&self, visit: &mut TagVisitor<'_>) {
+        for id in &self.exposed {
```

Comment:
Please add a comment explaining why we can skip `int_to_ptr_map` and `base_addr`.